actioncable/README.md

```diff
@@ -13,7 +13,7 @@ You can read more about Action Cable in the [Action Cable Overview](https://edge
 
 API documentation is at:
 
-* http://api.rubyonrails.org
+* https://api.rubyonrails.org/
```

Comment:
https://github.com/rails/rails/pull/35529#discussion_r263992781

Reply:
My Bad. Fixed it now.